src/test/java/steps/CheckboxSteps.java

```diff
@@ -32,44 +33,30 @@ public void user_is_on_checkbox_page(){
 
     /* TO DO */
     @When("User clicks checkbox")
-    public void user_clicks_checkbox(DataTable table) {
+    public void user_clicks_checkbox(DataTable table) throws Exception {
 
-        List<Map<String, Integer>> values = table.asMaps(String.class, Integer.class);
+        List<String> values = table.asList(String.class);
 
-        for(Map<String, Integer> input : values){
+        checkboxPage.fillElement(values.get(0), "");
```

Comment:
instead of         checkboxPage.fillElement(values.get(0), ""); you want         checkboxPage.fillElement(values.get(0)get(0), values.get(0)get(1));